src/ferui-components/tree-view/tree-view-component.ts

```diff
@@ -247,9 +269,19 @@ export class FuiTreeViewComponent<T> implements OnInit, OnDestroy {
    */
   private updateScrollerWidth(): void {
     let max = this.treeViewUtils.virtualScrollerWidth;
-    // takes into consideration padding
-    max = max > this.originalWidth ? (this.border ? max : max + 20) : this.border ? this.originalWidth : this.originalWidth - 20;
+    // takes into consideration 20px or the difference for padding
+    const diff = Math.abs(this.originalWidth - max);
+    const padding = diff !== 0 ? (diff > 20 ? 20 : diff) : 20;
```

Comment:
What is this 20px for ? If this value correspond to something specific you should create a variable and add a comment explaining what it is. And if this value is shared on multiple places, then you should create a constant that you use everywhere. I saw that you've used this 20px value on multiple places but i don't see what it is for.